ga4gh/datamodel/datasets.py

```diff
@@ -97,30 +94,32 @@ class SimulatedDataset(AbstractDataset):
     A simulated dataset
     """
     def __init__(
-            self, datasetId, randomSeed, numCalls,
-            variantDensity, numVariantSets, numAlignments):
-        super(SimulatedDataset, self).__init__()
-        self._id = datasetId
+            self, localId, randomSeed=1, numCalls=1,
+            variantDensity=1, numVariantSets=1, numAlignments=1):
+        super(SimulatedDataset, self).__init__(localId)
         self._randomSeed = randomSeed
         self._randomGenerator = random.Random()
         self._randomGenerator.seed(self._randomSeed)
 
+        # We do the imports here because we have circular dependencies
+        # otherwise.
+        import ga4gh.datamodel.variants as variants
```

Comment:
ew ew ew ew ew ew!  Let's move all the CompoundId subclasses to datamodel/__init__.py